WORKSPACE

```diff
@@ -20,7 +20,8 @@ load("//tensorflow:workspace.bzl", "tf_workspace")
 #android_sdk_repository(
 #    name = "androidsdk",
 #    api_level = 23,
-#    build_tools_version = "25.0.1",
+#    build_tools_version = "25.0.2",
+#    #Check your build tools version. Android Studio is shipping with 25.0.2 currently.
```

Comment:
After updating I only see 25.0.2 in AS within SDK manager and within my files on this ubuntu version of AS. I like your rephrasing as it allows for those differences.